data/json/items/gun/84x246mm.json

```diff
@@ -32,33 +33,34 @@
     "type": "GUN",
     "copy-from": "m3_carlgustav",
     "name": { "str": "M3E1 recoilless rifle" },
+    "description": "The Carl Gustaf M3E1 is a breech-loading 84 millimeter man-portable, reusable, multi-role recoilless rifle.  This is a shorter, lighter, and more advanced development of the M3.  Under normal circumstances, users should refrain from shooting more than 6 rounds per day to prevent undue risk of brain injury...",
```

Comment:
```suggestion
    "description": "The Carl Gustaf M3E1 is a breech-loading 84 millimeter man-portable, reusable, multi-role recoilless rifle.  This is a shorter, lighter, and more advanced development of the M3.  Under normal circumstances, users should refrain from shooting more than 6 rounds per day to prevent undue risk of brain injury…",
```